team-manager/src/main/java/ua/knu/knudev/teammanager/mapper/SpecialtyMapper.java

```diff
@@ -0,0 +1,10 @@
+package ua.knu.knudev.teammanager.mapper;
+
+import org.mapstruct.Mapper;
+import ua.knu.knudev.knudevcommon.mapper.BaseMapper;
+import ua.knu.knudev.teammanager.domain.Specialty;
+import ua.knu.knudev.teammanagerapi.dto.SpecialtyCreationDto;
+
+@Mapper(componentModel = "spring")
+public interface SpecialtyMapper extends BaseMapper<Specialty, SpecialtyCreationDto> {
```

Comment:
SpecialtyCreationDto is not the DTO. Either build domain by hand, or put here Dto. But it must be something like 
public interface SpecialtyMapper extends BaseMapper<Specialty, SpecialtyDto>

Reply:
done